lib/dynamo.js

```diff
@@ -22,8 +22,20 @@ function createClient(config) {
   }
 
   const client = new AWS.DynamoDB.DocumentClient(config);
+  client.consistentReads = 'consistentReads' in config && config.consistentReads;
   Promise.promisifyAll(client);
 
+  // patch read methods to use consistent read if specified
+  _.forEach(['queryAsync', 'getAsync', 'scanAsync'], function(fname) {
+    var tmpFunc = client[fname];
+    client[fname] = function wrapper(params) {
+      if (this.consistentReads && (fname != 'queryAsync' || !('IndexName' in params))) {
+        params.ConsistentRead = true;
```

Comment:
do we have to pay more for this?

Reply:
Not directly, but it does consume more RCU.  We're only talking about turning this on in for tests though, not across the board, see:

https://github.com/SparkPost/accusers-api/pull/288